Budget Blocks/Budget Blocks/Networking/Bearer.swift

```diff
@@ -8,7 +8,7 @@
 
 import Foundation
 
-struct Bearer {
+struct Bearer: Codable {
```

Comment:
Can we add a comment here about not using Codable but leaving it in case we want to refactor later?